src/zmqsubscriber/zmqsubscriber_test.go

```diff
@@ -43,22 +50,21 @@ func getTestZMQSubscriber(t *testing.T, env test.TestEnv) *ZMQSubscriber {
 }
 
 func TestZMQSubscriber(t *testing.T) {
+	const waitTimeout = 3 * time.Second
```

Comment:
his can be a toplevel const instead

Reply:
I'd like to have as few things as possible as global. Can move it once really needed.